xrviz/dashboard.py

```diff
@@ -20,23 +20,28 @@
 class Dashboard(SigSlot):
     """
     This section provides access to the complete generated dashboard,
-    consisting of all subsections.
+    consisting of all the panes.
 
     Parameters
     ----------
-    data: `xarray` instance: `DataSet` or `DataArray`
-           datset is used to initialize.
-    initial_params: To pre-set values of widgets.
+    data: `xr.core.dataarray.DataWithCoords`
+        Data is required for initialization.
+    initial_params: `dict`
+        To pre-select values of widgets upon initialization.
```

Comment:
Where can we find out about the keys and values allowed here?

Reply:
Added link to `Set Initial Parameters` doc.